lib/ip_wrangler/main.rb

```diff
@@ -1,26 +1,22 @@
-$config = YAML.load_file('config.yml')
+$config = YAML.load_file('./config.yml')
 
 use Rack::Auth::Basic, 'Restricted Area' do |username, password|
-  [username, password] == [$config[:username], $config[:password]]
+  [username, password] == [$config['username'], $config['password']]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.